### PR TITLE
Pritheeroy/slurm

### PR DESCRIPTION
**Pull request by pritheeroy**
This merge request centers around documentation for getting started with SLURM.

### CODE REVIEW

**Review by dravgav — COMMENTED** [review]
This looks great! I added a couple small comments on areas I thought could be improved, otherwise this is a very comprehensive, informative, and wellrounded article on SLURM.

File: Topics/Tech_Stacks/SLURM.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+### Troubleshooting
+
+- If nodes appear down or drained, check logs (typically under `/var/log/slurm/`) for errors.
+- Ensure network connectivity and correct configuration of `munge` for authentication issues.
+- For any issues with starting jobs, verify resource allocation and partition configurations.
+
+Remember, each HPC environment is unique, and specific configuration details might differ. It's essential to collaborate with your system administrator and refer to SLURM's comprehensive documentation for environment-specific adjustments.
+
+## Advanced SLURM Features
```
[the remark this answers is not in the document]
Comment (pritheeroy) [reply Topics/Tech_Stacks/SLURM.md:95]: Sounds good, let me elaborate on some of these advanced features.

File: Topics/Tech_Stacks/SLURM.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+- **Diagnose with `sinfo` and `squeue`**: Check the status of nodes and the job queue for issues.
+- **Review Resource Requests**: Ensure your job's resource requests are within the cluster's limits.
+- **Consult Cluster Policies**: Be aware of cluster-specific policies or limits set by administrators.
+
+### Debugging and Log Analysis
+
+- **SLURM Logs**: Check the SLURM logs (`/var/log/slurm/`) for cluster-wide issues.
+- **Job-Specific Logs**: Review stdout and stderr logs of your job for application-specific messages.
+- **Correlating Logs**: Compare timestamps between different logs for insights into complex issues.
+
```
[the remark this answers is not in the document]
Comment (dravgav) [reply Topics/Tech_Stacks/SLURM.md:211]: Great informative section!  I like how you included a link to the official website in order to check for new releases.

File: Topics/Tech_Stacks/SLURM.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dravgav) [reply Topics/Tech_Stacks/SLURM.md:96]: Great elaboration on these advanced features. I like how each feature had its own mini-section with additional details and a summary above.

File: Topics/Tech_Stacks/SLURM.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (pritheeroy) [reply Topics/Tech_Stacks/SLURM.md:211]: For sure, the best place to learn about Slurm is none other than the official Slurm website!

**Pull request merged**